participant/app/components/List.tsx

```diff
@@ -0,0 +1,30 @@
+import type { ReactElement } from "react";
+import type { i18nKey } from "app/types";
+import { Trans } from "react-i18next";
+
+export type ListProps = {
+  listKeys: i18nKey[];
+  ordered?: boolean;
```

Comment:
i kinda wish we were using an enum type here but 🤷🏻‍♀️
```suggestion
  type?: "ordered" || "unordered";
```

Reply:
Oh sure. I feel relatively neutral on this, so I'll change it.